python/pip_install/requirements.bzl

```diff
@@ -60,9 +60,11 @@ def compile_pip_requirements(
     deps = [
         requirement("click"),
         requirement("colorama"),
+        requirement("pep517"),
         requirement("pip"),
         requirement("pip_tools"),
         requirement("setuptools"),
+        requirement("tomli"),
```

Comment:
How it this one used by pip-tools. I checked the install-requires of pip-tools and it isn't there.

Reply:
`# via pep517`

Reply:
Ah cheers. Didn't expect `pep517` to take any deps.